src/scripts/main.js

```diff
@@ -2,7 +2,21 @@
 
 const people = require('./lib/people');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const table = document.querySelector('.dashboard');
 
-// write your code here
+for (const person of people) {
+  const row = document.createElement('tr');
+
+  createCell(person, row);
+
+  table.append(row);
+}
+
+function createCell(person, row) {
```

Comment:
Тоді ти створюєш клітинки
```suggestion
function createCells(person, row) {
```

Reply:
Я би радив створювати td елемент
`сreateCell(person.sex === 'm' ? 'Male' : 'Female', row)`
`createCell(person.died - person.born, row)`

і так далі